### DISCUSSION TITLE
Github pages

### DISCUSSION THREAD

**Discussion by Christian202120792** [community/community#61501] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello, i'm new and i have a question with the pages of github, speficically with the name before the .github.io, the name is necessarily the username? or can i choose other name?

**Thanks for you time**

**Comment by abinth11** [comment, marked as answer]
Hello there,

Welcome to GitHub! I'm happy to help you with your question.

When creating a GitHub Pages website, the name before the `.github.io` in the URL does not have to be your GitHub username. GitHub allows you to choose a custom name for your Pages website, but it must be unique within the GitHub Pages domain.

Here's how you can select a custom name for your GitHub Pages website:

1. Navigate to your GitHub repository that contains the content you want to publish as a GitHub Pages site.

2. Click on the "Settings" tab located near the top of your repository.

3. Scroll down to the "GitHub Pages" section.

4. Under the "Source" dropdown, select the branch that contains the content you want to publish. By default, it is usually set to the "main" branch.

5. Next to the branch selection, you will see an option to choose a custom domain. Here, you can enter a unique name for your GitHub Pages website. It can be your username, a project name, or anything else you prefer (as long as it's available).

6. Click on the "Save" button to apply the changes.

Once you've saved the custom domain, your GitHub Pages website will be accessible at `https://your-custom-name.github.io`.

Keep in mind that it may take a few minutes for the changes to propagate, so be patient if the URL doesn't work immediately after making the updates.

**Reply by Christian202120792** [reply]
Really thanks you very much, it's much help

**Reply by ericklove8e5** [reply]
Go ahead ❤️love

**Comment by ericklove8e5** [comment]
christian202120792

**Comment by Kibreab-kb-kb** [comment]
On GitHub, the name before the .github.io in the URL of your GitHub Pages is typically associated with your username. However, you can choose a custom name for your GitHub Pages by creating a repository with that custom name and enabling GitHub Pages for that repository. This way, your GitHub Pages site will be accessible at "customname.github.io" instead of "username.github.io". Keep in mind that the custom name must still be unique across all GitHub Pages sites.

**Comment by xexexex-cell** [comment]
hello

I have a question, after publishing my landing page, images and styles are not displayed on the GitHub page. I still did not understand what the problem was

https://github.com/xexex-cell/kitchen.git

upd.
sorry, my bad)

**Reply by Kibreab-kb-kb** [reply]
It sounds like there might be an issue with the file paths or links to your images and stylesheets in the HTML/CSS code. Make sure the paths are correct and that you've pushed all the necessary files to your GitHub repository. Double-check the file structure and links to ensure everything is properly connected.
